concordia/version.py

```diff
@@ -0,0 +1,7 @@
+from setuptools_scm import get_version
+import functools
+
+
+@functools.lru_cache(maxsize=None)
```

Comment:
I'm not sure maxsize matters since we have no arguments but it also doesn't matter